setup.py

```diff
@@ -120,6 +120,7 @@
         'scripts/decode-syseeprom',
         'scripts/dropcheck',
         'scripts/disk_check.py',
+        'scripts/dpu_tty.py',
```

Comment:
Can you please rename the application to `dpu-tty`? To make the user experience more convenient

Reply:
Changed